ci/jenkins/test-vmc.sh

```diff
@@ -328,7 +328,7 @@ function run_codecov { (set -e
 
     if [[ $remote == true ]]; then
         ${SCP_WITH_UTILS_KEY} codecov jenkins@${ip}:~
-        ${SSH_WITH_UTILS_KEY} -n jenkins@${ip} "~/codecov -c -t ${CODECOV_TOKEN} -F ${flag} -f ${file} -C ${GIT_COMMIT} -r antrea-io/antrea"
+        ${SSH_WITH_UTILS_KEY} -n jenkins@${ip} "cd antrea; ~/codecov -c -t ${CODECOV_TOKEN} -F ${flag} -f ${file} -C ${GIT_COMMIT} -r antrea-io/antrea"
```

Comment:
`-s` does not work?

Reply:
I didn't try `-s`, let me update to see if it also works.

Reply:
Look like `-s` doesn't work in this case, we need to run codecov in side of Antrea project root to fix the relative path issue in the coverage raw data.